bin/system-integrity

```diff
@@ -0,0 +1,263 @@
rpm_checks":"??5??????","type":"integrityissue"}],"kano-settings":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/polkit-1/localauthority/50-local.d/10-udisks.pkla","raw_rpm_checks":"??5??????","type":"integrityissue"},{"failed_md5sum":true,"is_conf":false,"path":"/usr/share/kano-settings/media/Parental/parental-hosts-blacklist.gz","raw_rpm_checks":"??5??????","type":"integrityissue"}],"kano-toolset":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/network/interfaces","raw_rpm_checks":"??5??????","type":"integrityissue"}],"kano-video":[{"failed_md5sum":true,"is_conf":false,"path":"/usr/share/applications/auto_video-cli.desktop","raw_rpm_checks":"??5??????","type":"integrityissue"}],"lightdm":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/lightdm/lightdm.conf","raw_rpm_checks":"??5??????","type":"integrityissue"}],"login":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/pam.d/login","raw_rpm_checks":"??5??????","type":"integrityissue"}],"make-light":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/sudoers.d/make-light-powerup_conf","raw_rpm_checks":"??5??????","type":"integrityissue"},{"failed_md5sum":true,"is_conf":true,"path":"/etc/sudoers.d/make-light_conf","raw_rpm_checks":"??5??????","type":"integrityissue"}],"make-music":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/sudoers.d/make_music_conf","raw_rpm_checks":"??5??????","type":"integrityissue"}],"openbox":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/xdg/openbox/rc.xml","raw_rpm_checks":"??5??????","type":"integrityissue"}],"openbox-lxde-session":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/xdg/lxsession/LXDE/autostart","raw_rpm_checks":"??5??????","type":"integrityissue"},{"failed_md5sum":true,"is_conf":true,"path":"/etc/xdg/openbox/LXDE/rc.xml","raw_rpm_checks":"??5??????","type":"integrityissue"}],"passwd":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/default/useradd","raw_rpm_checks":"??5??????","type":"integrityissue"}],"preload":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/preload.conf","raw_rpm_checks":"??5??????","type":"integrityissue"}],"procps":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/sysctl.conf","raw_rpm_checks":"??5??????","type":"integrityissue"}],"raspberrypi-sys-mods":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/sudoers.d/010_pi-nopasswd","raw_rpm_checks":"??5??????","type":"integrityissue"}],"sudo":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/sudoers","raw_rpm_checks":"??5??????","type":"integrityissue"},{"failed_md5sum":true,"is_conf":true,"path":"/etc/sudoers.d/README","raw_rpm_checks":"??5??????","type":"integrityissue"}],"systemd":[{"failed_md5sum":true,"is_conf":true,"path":"/etc/systemd/system.conf","raw_rpm_checks":"??5??????","type":"integrityissue"}],"wpasupplicant":[{"failed_md5sum":true,"is_conf":false,"path":"/sbin/wpa_cli","raw_rpm_checks":"??5??????","type":"integrityissue"},{"failed_md5sum":true,"is_conf":false,"path":"/sbin/wpa_supplicant","raw_rpm_checks":"??5??????","type":"integrityissue"}]}')
```

Comment:
Hmm, this is a bit weird - you are parsing a JSON which is essentially a Python `dict()` so at a minimum set this variable directly. Ideally you would include another file which is just the `known_issues.json` file and load it in at runtime - you could even provide a CLI argument to provide the known issues file with a default value of the location of this JSON.

Reply:
Agreed on that one too - how about something like `/etc/system-integrity.json` ?

Reply:
@tombettany @skarbat The reason for all these was simplicity in the future.

 - Here's the use case: You get reports of issues during img builds. You release and want to update the JSON. You'd only have to paste it in rather than faffing about to change it.
 - It's a JSON because it's more standardised. 
 - It's also inlined because I didn't want to scroll for hundreds of lines before I get to the code.
 - Also, I didn't want to add extra files and complexity beyond that because We don't yet know how this is going to evolve in time. It's the first MVP.

Reply:
Okay, if you think that this is best then let's go with it.

Reply:
Just a quick idea here (I want to write it down in a clearer way, probably in an RFC) that in the future we could utilise a hidden directory like by default Linux distros do with `~/.config` in order to put configuration files [there are already a few but we can standardise this] and something like `~/.settings` for other files.

I agree with @radujipa that we shouldn't add random files around but also it's a good point of @tombettany to have the `json` in a seperate file.

Reply:
Your wishes have been fulfilled. There is now a separate file. 🎉